### DISCUSSION TITLE
GitHub List Error

### DISCUSSION THREAD

**Discussion by 0xvashishth** [community/community#54245] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

There is no redirection or removing modal after creating the list. It is just showing `Creating...`
Below is the bug screenshot.
![13](https://user-images.githubusercontent.com/76911582/235655325-32353965-9ea5-4460-b98a-d9174c4d1611.JPG)

And After clicking on `Share Feedback and report bugs` the modal is still not disappearing..
Below is the bug screenshot.
![14](https://user-images.githubusercontent.com/76911582/235655573-8f4df7b1-5fb0-41ad-bf41-00552ac4f17f.JPG)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @vasu-1 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by 0xvashishth** [reply]
Hey 👋
Thanks for the correction!

**Comment by Jamim** [comment]
This discussion duplicates #50880, just as a dozen of other reports.

**Comment by jerofasion** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Select Feature Area
> 
> - [X] Issues
> - [ ] Projects
> 
> ### Body
> 
> There is no redirection or removing modal after creating the list. It is just showing `Creating...`
> Below is the bug screenshot.
> ![13](https://user-images.githubusercontent.com/76911582/235655325-32353965-9ea5-4460-b98a-d9174c4d1611.JPG)
> 
> And After clicking on `Share Feedback and report bugs` the modal is still not disappearing..
> Below is the bug screenshot.
> ![14](https://user-images.githubusercontent.com/76911582/235655573-8f4df7b1-5fb0-41ad-bf41-00552ac4f17f.JPG)
> 
> 
> ### Guidelines
> 
> - [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by AMU0704862105** [comment]
[]()

**Reply by jerofasion** [reply]
Not problem

**Comment by Adv34ot** [comment]
Салам алейкум
